.github/workflows/frontend-integration-test.yml

```diff
@@ -59,8 +59,7 @@ jobs:
     strategy:
       matrix:
         architecture: [linux/amd64]
-        java-version: [ 8, 11, 17 ]
-        test-mode: [ embedded, deploy ]
+        java-version: [ 8 ]
```

Comment:
Why do we only check Java 8 and remove the rest here?

Reply:
The frontend testing part doesn't need to test the Java compilation part, so only need one Java version.